script/travis/travis_retry.sh

```diff
@@ -1,25 +1,5 @@
 #!/bin/bash
 
-#
-# Copyright 2017 Benjamin Worpitz
```

Comment:
Why do you removed the license header?

Reply:
I removed the copyright header because this code is not from me but a minimally adapted version of some travis code. So we should not reserve copyright.